tool/tctl/common/tctl.go

```diff
@@ -93,11 +59,11 @@ type GlobalCLIFlags struct {
 type CLICommand interface {
 	// Initialize allows a caller-defined command to plug itself into CLI
 	// argument parsing
-	Initialize(*kingpin.Application, *servicecfg.Config)
+	Initialize(*kingpin.Application, *tctlcfg.GlobalCLIFlags, *servicecfg.Config)
 
 	// TryRun is executed after the CLI parsing is done. The command must
 	// determine if selectedCommand belongs to it and return match=true
-	TryRun(ctx context.Context, selectedCommand string, c *authclient.Client) (match bool, err error)
+	TryRun(ctx context.Context, cmd string, clientFunc commonclient.InitFunc) (match bool, err error)
```

Comment:
This broke e/ due to the signature change.

Reply:
should have updated `teleport.e` reference after merging https://github.com/gravitational/teleport.e/pull/5851